Xamarin.Forms.Platform.Android/Renderers/BottomNavigationViewUtils.cs

```diff
@@ -20,6 +30,162 @@ namespace Xamarin.Forms.Platform.Android
 {
 	public static class BottomNavigationViewUtils
 	{
+
+		internal const int MoreTabId = 99;
```

Comment:
Is there any guarantee that this Id will not be occupied by another View?
Maybe better generate unique Id immediately before create MoreTabView.

Reply:
This should be fine. The menu items are added based on the indexes of the page in the TabPage. The max value we can use here is 99 so there's always a small chance of collision if developers are adding their own menu with Id numbers. But this is also new behavior so developers shouldn't really have scenarios that currently exist where they are adding > 5 items to a bottom navigation bar